install/generator/04-syndesis-db-metrics.yml.mustache

```diff
@@ -1,122 +0,0 @@
-- apiVersion: v1
-  kind: ConfigMap
-  metadata:
-    name: syndesis-db-metrics-config
-    labels:
-      app: syndesis
-      syndesis.io/app: syndesis
-      syndesis.io/type: infrastructure
-      syndesis.io/component: syndesis-db-metrics
-  data:
-    queries.yaml: |
-        pg_database:
-          query: " SELECT pg_database.datname, pg_database_size(pg_database.datname) as size FROM pg_database"
-          metrics:
-            - datname:
-                usage: "LABEL"
-                description: "Name of the database"
-            - size:
-                usage: "GAUGE"
-                description: "Disk space used by the database"
-- apiVersion: v1
-  kind: Service
-  metadata:
-    labels:
-      app: syndesis
-      syndesis.io/app: syndesis
-      syndesis.io/type: infrastructure
-      syndesis.io/component: syndesis-db-metrics
-    name: syndesis-db-metrics
```

Comment:
We need to carry this service over into `syndesis-db.yml.mustache`. Else the Prometheus ServiceMonitor will have no endpoint to scrape.

Or we expose 9187 on the existing syndesis-db service. But, `install/addons/syndesis-db-servicemonitor.yml` would need modifying if we do that.